extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -162,10 +166,21 @@ export class AmpAnalytics extends AMP.BaseElement {
   layoutCallback() {
     // Now that we are rendered, stop rendering the element to reduce
     // resource consumption.
+    this.element.getResources().setOwner(this.element,
```

Comment:
Just reproduced. The call stack is:
![image](https://user-images.githubusercontent.com/2455047/28391551-7bf01e5a-6cac-11e7-946f-5ecaeaddccb1.png)